web/packages/teleport/src/Discover/Kubernetes/EnrollEKSCluster/EnrollEksCluster.tsx

```diff
@@ -93,10 +107,18 @@ export function EnrollEksCluster(props: AgentStepProps) {
       status: 'notStarted',
     });
   const [isAppDiscoveryEnabled, setAppDiscoveryEnabled] = useState(true);
+  const [isAutoDiscoveryEnabled, setAutoDiscoveryEnabled] = useState(true);
   const [isAgentWaitingDialogShown, setIsAgentWaitingDialogShown] =
     useState(false);
   const [isManualHelmDialogShown, setIsManualHelmDialogShown] = useState(false);
   const [waitingResourceId, setWaitingResourceId] = useState('');
+  const [discoveryGroupName, setDiscoveryGroupName] = useState(() =>
+    cfg.isCloud ? DISCOVERY_GROUP_CLOUD : DEFAULT_DISCOVERY_GROUP_NON_CLOUD
```

Comment:
i'd leave it empty string if cloud, to make a point that that variable only applies to self-hosted

Reply:
Done.